chpl/chpl-service/src/main/java/gov/healthit/chpl/sharedstore/SharedStorePrimaryKey.java

```diff
@@ -0,0 +1,53 @@
+package gov.healthit.chpl.sharedstore;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Embeddable
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class SharedStorePrimaryKey implements Serializable {
+    private static final long serialVersionUID = 8887068260992258871L;
+
+    @Column(name = "domain")
+    private String domain;
+
+    @Column(name = "key")
+    private String key;
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + Objects.hash(key, domain);
```

Comment:
That is the code generated by Eclipse.  Not sure why...  Also, wondering if I should just use Lombok equals and hashcode....

Reply:
I would vote to not use the Lombok ones because this is related to the Entity and we have had problems with the Lombok stuff in entities. I'm fine with this generated code.